tools/Get-NuGetTool.ps1

```diff
@@ -0,0 +1,15 @@
+<#
```

Comment:
The reason I prefer powershell is it's already on every Windows machine. And this file doesn't require any maintenance other than (someday) rev'ing the "3.4.4" version. So I'd hesitate to introduce any cakebuild dependencies that means I can't build the repo without installing a 3rd party dependency.